### PR TITLE
Revert "bpo-36543: Remove the xml.etree.cElementTree module."

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
Reverts python/cpython#19108

https://bugs.python.org/issue36543

### CODE REVIEW

**Comment by scoder** [comment]
Thanks Serhiy. Was there a News entry and docs update that went with the removal? Those would then need to be reverted as well.

**Review by scoder — APPROVED** [review]

File: Misc/NEWS.d/3.9.0a1.rst [hunk 1]
```diff
@@ -3372,7 +3372,7 @@ markup and any values in the message. Patch by Paul Ganssle
 .. section: Library
 
 Removed methods Element.getchildren(), Element.getiterator() and
-ElementTree.getiterator() and the xml.etree.cElementTree module.
```
Comment (tiran) [inline Misc/NEWS.d/3.9.0a1.rst]: Don't remove news entries from published releases. For users, packagers and distributions it's important to know how the Python has changed between alpha releases.

**Review by tiran — CHANGES_REQUESTED** [review]
I'd still prefer to land my PR #19921 and mark the module for removal in 3.10.

**Comment by serhiy-storchaka** [comment]
I messed this PR. Recreated as #20117.

**Pull request closed without merging**